libr/hash/luhn.h

```diff
@@ -0,0 +1,6 @@
+#ifndef _LUHN_H_
```

Comment:
breaks standard include barrier naming rules. it shouldnt start by underline (read it somewhere in some good practices book some day)